robots/car_display.h

```diff
@@ -107,6 +107,64 @@ class CarDisplay
         return out;
     }
 
+    std::vector<float> getMouseClickPixelPosition() const
```

Comment:
It'd probably be better to return a ``std::array<float,2>`` here

Reply:
:+1: